### PR TITLE
Center active page in pagiator

### PR DESCRIPTION
**Pull request by akempes**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Only the first 5 pages are shown, to get to page 6 you need to select page 5 and then click on the 'next page' arrow.

**What is the new behavior?**
The active page is centered in the pager. If page 6 is active, the pager will look like this:
screen shot 2017-07-17 at 13 30 30

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/components/footer/pager.component.ts [hunk 1]
```diff
@@ -136,8 +136,16 @@ export class DataTablePagerComponent {
...
 
     if (isMaxSized) {
-      startPage = ((Math.ceil(page / maxSize) - 1) * maxSize) + 1;
-      endPage = Math.min(startPage + maxSize - 1, this.totalPages);
+      startPage = page-Math.floor(maxSize/2);
+      endPage = page+Math.floor(maxSize/2), this.totalPages;
+
+      if(startPage <1){
+        startPage = 1;
+        endPage = startPage+maxSize-1;
```
Comment (amcdnl) [inline src/components/footer/pager.component.ts]: Can you add a space between the `+` and `-`

File: src/components/footer/pager.component.ts [hunk 2]
```diff
@@ -136,8 +136,16 @@ export class DataTablePagerComponent {
     page = page || this.page;
 
     if (isMaxSized) {
-      startPage = ((Math.ceil(page / maxSize) - 1) * maxSize) + 1;
-      endPage = Math.min(startPage + maxSize - 1, this.totalPages);
+      startPage = page-Math.floor(maxSize/2);
```
Comment (amcdnl) [inline src/components/footer/pager.component.ts]: Can you add a space between `maxSize / 2`

**Comment by akempes** [comment]
Hello,

I've just pushed an update.

Thanks!


Kindest regards, Arjan Kempes

Met vriendelijke groet, Arjan Kempes

*CREATIVE SKILLS*
ARJAN KEMPES WEB DEVELOPER
CREATIVE SKILLS, VEERDIJK 42R, 1531 MS, WORMER
T: 075 647 69 40 F: 075 647 69 41

2017-07-17 17:55 GMT+02:00 Austin <notifications@github.com>:

**Comment by madhuinwiter** [comment]
@amcdnl and @akempes because of the abave changes in pagination.component.ts pagintion display is broken and few tests are getting failed when I raised PR. Please take a look. I am tested by replacing the old code it worked fine. Please find the attached screenshot for the broken display.

ngx-datatable pagination broken

Thank you.

**Comment by madhuinwiter** [comment]
Guys, I fixed this bug in my PR #887 .

**Pull request merged**